### PR TITLE
Add Kramdown and Notes

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: STYLE.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ We'll be using a subset (TBD) of the full [list of checks][checks].
 
 [checks]: https://github.com/coala/bear-docs/blob/master/docs/MarkdownBear.rst#settings "MarkdownBear checks"
 
+We use Kramdown, with GFM codeblocks, and a few Kramdown related exceptions introduced with `{: }` such as class names for specific formatting of columns and notes.
```
Comment (BanzaiMan) [inline STYLE.md]: What's GFM?

Comment (plaindocs) [inline STYLE.md]:
```suggestion
We use Kramdown, with [GFM](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) codeblocks, and a few Kramdown related exceptions introduced with `{: }` such as class names for specific formatting of columns and notes.
```

Comment (plaindocs) [reply STYLE.md]: Thanks for asking!

**Review by c3333 — COMMENTED** [review]
Lernen danke

**Pull request merged**